library/L2_HAL/sensors/environment/light.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "utility/units.hpp"
+
+namespace sjsu
+{
+class Light
```

Comment:
```suggestion
class LightSensor
```
Because Light makes it sound like we are controlling a lightbulb.